tests/Unit/inc/ThirdParty/Plugins/CDN/Cloudflare/displayServerPushingModeNotice.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\ThirdParty\Plugins\CDN\Cloudflare;
+
+use Mockery;
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Unit\TestCase;
+use WP_Rocket\Admin\Options_Data;
+use WP_Rocket\ThirdParty\Plugins\CDN\Cloudflare;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Plugins\CDN\Cloudflare::display_server_pushing_mode_notice
+ *
+ * @group  ThirdParty
+ * @uses   ::rocket_get_constant
+ */
+class Test_DisplayServerPushingModeNotice extends TestCase{
+    private $options;
+    private $cloudflare;
+
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->options  = Mockery::mock( Options_Data::class );
+        $this->cloudflare = new Cloudflare( $this->options );
+
+        $this->stubTranslationFunctions();
+    }
+
+    /**
+     * @dataProvider configTestData
+     */
+    public function testShouldReturnExpected( $config, $expected ) {
+
+        Functions\stubs([
+            'get_current_screen'   => $config['current_screen'],
+            'current_user_can'    => $config['capability'],
+            'rocket_get_constant' => $config['server_push'],
```

Comment:
Instead of stubbing `rocket_get_constant`, you can use `$this->constants['CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE'] = $config['server_push']` at the start of the test method, it will be automatically manage by the StubTrait that way